Source/UZKArchive.m

```diff
@@ -866,7 +870,7 @@ - (BOOL)extractBufferedDataFromFile:(NSString *)filePath
         
         long long bytesDecompressed = 0;
         
-        NSError *strongInnerError = nil;
+        int innerErr = 0;
```

Comment:
@amosavian As you can see, I already fixed this bug a different way in the `v1.9` branch. We'll roll back to that.

Reply:
I'm not an objc expert. I could see that autoreleasepool would deallocate objects (including `NSError`). We can rollback if you have a better solution to exclude error object from autoreleaspool.